src/RadioGroup/RadioGroup.tsx

```diff
@@ -15,6 +15,7 @@ interface Option<IDType extends string = string, ValueType = any> {
   /** A BCP47 language tag. */
   lang?: string;
   value?: ValueType;
+  additionalText?: string;
```

Comment:
Argh not sure what the best name for this field is. `textBelowSelectedOption`? `additionalTextIfSelected`?

Reply:
either of those options work for me!

Reply:
also just plain `additionalText` works!